scripts/batch_mr_graphics_bench.py

```
@@ -0,0 +1,520 @@
+                print(" ".join(shlex.quote(x) for x in argv))
+                continue
+
+            print(
+                f"[{idx}/{total_scenes}] run: {scene_name} ({model.name}), "
+                f"warmup={warmup}, measure={measure}, please wait",
+                file=sys.stderr,
+            )
+            stats_path.unlink(missing_ok=True)
+
+            try:
+                proc = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)